lib/stats/index.js

```diff
@@ -0,0 +1,10 @@
+module.exports = exports;
```

Comment:
Also This file is not needed and could be an unnecessary work to maintain.  It doesn't _add_ any value. I think direct require calls would be better.